src/components/KButton/KButton.vue

```diff
@@ -163,12 +163,25 @@ export default defineComponent({
       return ''
     })
 
+    /**
+     * Strips falsy `disabled` attribute, so it does not fall onto native <a> elements
+     */
+    const strippedAttrs = computed((): typeof attrs => {
+      const { disabled, ...restAttrs } = attrs
+      if (disabled) {
+        return attrs
```

Comment:
suggestion: can you add a comment here explaining why we are modifying the disabled attr?

Reply:
Sure

Reply:
Comment added.